converter/internal/prometheusconvert/prometheusconvert_test.go

```diff
@@ -44,6 +44,11 @@ func TestConvert(t *testing.T) {
 					}
 				}
 
+				// If we expect more errors than we got
+				if len(expectedErrors) > len(diags) {
```

Comment:
Should this be a `!=` ? Or is diags being less the only valid use case? Would complicate your fails a bit though.

Reply:
the `<` (opposite) case is handled above on line 43